misc/dbt-materialize/tests/adapter/test_clusters.py

```diff
@@ -100,7 +100,7 @@ def test_materialize_override_ok(self, project):
         results = run_dbt(["seed"])
         assert len(results) == 1
 
-        project.run_sql("CREATE CLUSTER not_default REPLICAS (r1 (SIZE '1'))")
+        project.run_sql("CREATE CLUSTER not_default SIZE = '1'")
```

Comment:
```suggestion
        project.run_sql("CREATE CLUSTER not_default SIZE = '3xsmall'")
```

Reply:
This is something for using the image which has different values than cloud (and is really only an internal setting so we don't document it).

Reply:
Oops. My apologies, and for the delayed review too.